src/Servers/HttpSys/test/NonHelixTests/DelegateOutOfProcTests.cs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Http;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Microsoft.AspNetCore.Testing;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Server.HttpSys.NonHelixTests
```

Comment:
Is there a reason why we don't want these to run on helix?

Reply:
We never figured out how to make them work there. https://github.com/dotnet/aspnetcore/issues/8247